analyzers/tests/SonarAnalyzer.UnitTest/Rules/UseReturnStatementTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+extern alias vbnet;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.UnitTest.TestFramework;
+using VisualBasic = vbnet::SonarAnalyzer.Rules.VisualBasic;
```

Comment:
We've got rid of of the `extern alias` statements in one of latest commits, as they are not needed. Let's keep things simple:
```suggestion
using Microsoft.VisualStudio.TestTools.UnitTesting;
using SonarAnalyzer.Rules.VisualBasic;
using SonarAnalyzer.UnitTest.TestFramework;
```
or like this to keep aligned with other UTs on current master:
```suggestion
using Microsoft.VisualStudio.TestTools.UnitTesting;
using SonarAnalyzer.UnitTest.TestFramework;
using VB = SonarAnalyzer.Rules.VisualBasic;
```